can_tools/scrapers/official/federal/CDC/cdc_testing_cases.py

```diff
@@ -0,0 +1,60 @@
+from typing import Dict
+
+import pandas as pd
+
+from can_tools.scrapers.base import CMU
+from can_tools.scrapers.official.base import FederalDashboard
+
+# Set the maximum number of records to ten million to ensure we collect all rows in the dataset.
+# Currently the size of the dataset is 2 million records, so this may need to be updated (increased)
+# in the future
+SODA_API_RESPONSE_LIMIT = 10000000
+
+
+class CDCHistoricalTestingDataset(FederalDashboard):
+    has_location = True
+    location_type = "county"
+    source = "https://data.cdc.gov/Public-Health-Surveillance/United-States-COVID-19-County-Level-of-Community-T/nra9-vzzn/data"
+    source_name = "Centers for Disease Control and Prevention"
+    provider = "cdc2"
+
+    # SODA API stream has different variable names than the CSV and online dataset -- but the columns appear to be the same.
+    variables = {
+        "cases_per_100k_7_day_count": CMU(
+            category="cases", measurement="rolling_average_7_day", unit="people"
+        ),
+        "percent_test_results_reported": CMU(
+            category="pcr_tests_positive",
+            measurement="rolling_average_7_day",
+            unit="percentage",
+        ),
+    }
+
+    def fetch(self) -> pd.DataFrame:
+        # select only the columns we care about in order to speed up query
+        data = pd.read_json(
+            f"https://data.cdc.gov/resource/nra9-vzzn.json?$limit={SODA_API_RESPONSE_LIMIT}"
+            "&$select=fips_code,date,cases_per_100k_7_day_count,percent_test_results_reported"
+        )
+        if len(data) == SODA_API_RESPONSE_LIMIT:
+            raise ValueError(
+                f"Size of the dataset has reached the currently set API response limit of {SODA_API_RESPONSE_LIMIT}. "
+                "Please increase the limit by modifying SODA_API_RESPONSE_LIMIT."
+            )
+        return data
+
+    def normalize(self, data: pd.DataFrame) -> pd.DataFrame:
+        return (
+            # remove suppressed values as we have no way to handle them
+            data.loc[~data.eq("suppressed").any(1)]
```

Comment:
Just for my own curiosity (I'm not too worried about it), does this mean that if either `cases_per_100k_7_day_count` or `percent_test_results_reported` is suppressed, then we will end up dropping the whole row (both variables)?

Reply:
ah, this would and it shouldn't -- good catch I'll fix this